2 - RTB/Documentazione esterna/Analisi dei Requisiti/Analisi dei Requisiti.typ

```diff
@@ -889,24 +906,20 @@ Dove:
     [*Codice*], [*Classificazione*], [*Descrizione*], [*Riferimento*],
     [FM-1], [Obbligatorio], [L'utente deve poter creare il magazzino.], [UC-1],
     [FM-1.1], [Obbligatorio], [L'utente deve poter caricare un file SVG contenente la pianta del magazzino.], [UC-1.1],
-    [FD-1.1.1], [Desiderabile], [L'utente deve poter definire le altezze degli elementi del file SVG tramite trascinamento verso l'alto.], [Verbale esterno 23-12-06],
-    [FM-1.1.2], [Obbligatorio], [L'utente visualizza un errore di importazione del file SVG.], [UC-1.1.1],
-    [FM-1.1.2.1], [Obbligatorio], [L'utente visualizza un errore dato dal caricamento di un file SVG privo di informazioni.], [UC-1.1.1.1],
-    [FM-1.1.2.2], [Obbligatorio], [L'utente visualizza un errore dato da informazioni incongruenti nel file SVG.], [UC-1.1.1.2],
-    [FM-1.2], [Obbligatorio], [L'utente deve poter creare manualmente il magazzino.], [UC-1.2],
-    [FM-1.2.1], [Obbligatorio], [L'utente deve poter inserire manualmente la lunghezza del magazzino.], [UC-1.2],
-    [FM-1.2.2], [Obbligatorio], [L'utente deve poter inserire manualmente la larghezza del magazzino.], [UC-1.2],
-    [FM-1.2.3], [Obbligatorio], [L'utente deve poter inserire manualmente l'altezza del magazzino.], [UC-1.2],
-    [FM-1.2.4], [Obbligatorio], [L'utente visualizza un errore dato dalla non validità dei dati inseriti.], [UC-1.2.1],
+    [FM-1.1.1], [Obbligatorio], [L'utente deve poter creare un magazzino tramite caricamento di un file SVG in qualunque momento.], [UC-1.1],
```

Comment:
"In qualunque momento" è troppo generico... è come dire che durante magari la configurazione del database io posso creare il magazzino caricando un file svg... o magari mentre ricerco un prodotto... io rimuoverei quella dicitura e semmai, come nel caso precedente, specificherei che la funzionalità rimane disponibile qualora si volesse ricreare o cambiare l'ambiente. Aggiornare ovviamente use case e immagine.